### PR TITLE
[Network]Bump to azure-mgmt-network 9.0.0

### LINKED ISSUE #11988: [Network] Fix actions required for az network private-link-service connection update and bump to 9.0.0
**Is your feature request related to a problem? Please describe.**

**Describe the solution you'd like**

**Describe alternatives you've considered**

**Additional context**

### PR DESCRIPTION
**Pull request by MyronFanQiu**
Fix #11988 

---

This checklist is used to make sure that common guidelines for a pull request are followed.

- [ ] The PR has modified HISTORY.rst describing any customer-facing, functional changes. Note that this does not include changes only to help content. (see [Modifying change log](https://github.com/Azure/azure-cli/tree/master/doc/authoring_command_modules#modify-change-log)).

- [ ] I adhere to the [Command Guidelines](https://github.com/Azure/azure-cli/blob/dev/doc/command_guidelines.md).

### CODE REVIEW

**Comment by yonzhan** [comment]
add to S165.

File: src/azure-cli/azure/cli/command_modules/network/custom.py [hunk 1]
```diff
@@ -77,7 +77,9 @@ def list_express_route_circuits(cmd, resource_group_name=None):
 
 def create_express_route_auth(cmd, resource_group_name, circuit_name, authorization_name):
     client = network_client_factory(cmd.cli_ctx).express_route_circuit_authorizations
-    return client.create_or_update(resource_group_name, circuit_name, authorization_name)
+    ExpressRouteCircuitAuthorization = cmd.get_models('ExpressRouteCircuitAuthorization')
```
Comment (MyronFanQiu) [inline src/azure-cli/azure/cli/command_modules/network/custom.py]: `ExpressRouteCircuitAuthorization` has two fields. I tried to expose these two in our command UX, however, during my test I cannot set any of them. The server will set both of two properties.

File: src/azure-cli/azure/cli/command_modules/network/custom.py [hunk 2]
```diff
@@ -2510,7 +2512,7 @@ def update_private_endpoint_connection(cmd, resource_group_name, service_name, p
     connection_state = PrivateLinkServiceConnectionState(
         status=connection_status,
         description=description,
-        action_required=action_required
+        actions_required=action_required
```
Comment (MyronFanQiu) [inline src/azure-cli/azure/cli/command_modules/network/custom.py]: This is the actual fix

File: src/azure-cli/azure/cli/command_modules/network/tests/latest/test_network_commands.py [hunk 3]
```diff
@@ -122,9 +122,10 @@ def test_network_private_endpoints(self, resource_group):
 
         pe_connection_name = self.cmd('network private-link-service show -g {rg} -n {lks1}').get_output_in_json()['privateEndpointConnections'][0]['name']
         self.kwargs['pe_connect'] = pe_connection_name
-        self.cmd('network private-link-service connection update -g {rg} -n {pe_connect} --service-name {lks1} --connection-status Rejected')
+        self.cmd('network private-link-service connection update -g {rg} -n {pe_connect} --service-name {lks1} --connection-status Rejected --action-required "need action"')
```
Comment (MyronFanQiu) [inline src/azure-cli/azure/cli/command_modules/network/tests/latest/test_network_commands.py]: this is the test

**Comment by haroldrandom** [comment]
Need to upgrade other related commands' YAML recording?

**Comment by MyronFanQiu** [comment]
@haroldrandom I didn't bump the api-version. so I don't think we should re-run the test.

**Comment by haroldrandom** [comment]
> we

But azure-mgmt-network==9.0.0 use API string 2019-11-01 while azure-mgmt-network==7.0.0 use 2019-09-01, Doesn't it matter?

**Comment by MyronFanQiu** [comment]
> > we
> 
> But azure-mgmt-network==9.0.0 use API string 2019-11-01 while azure-mgmt-network==7.0.0 use 2019-09-01, Doesn't it matter?

no. I'm still using 2019-09-01. I didn't bump the api-version

**Pull request closed without merging**